platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-dx010.init

```diff
@@ -33,45 +33,36 @@ case "$1" in
 start)
         echo -n "Setting up board... "
 
-        rmmod i2c-i801
-        rmmod i2c-ismt
         modprobe i2c-dev
-        modprobe i2c-i801
-        modprobe i2c-ismt
         modprobe i2c-mux-pca954x
         modprobe dx010_wdt
         modprobe leds-dx010
         modprobe lm75
 
-        found=0
-        for devnum in 0 1; do
-                devname=`cat /sys/bus/i2c/devices/i2c-${devnum}/name`
-                # iSMT adapter can be at either dffd0000 or dfff0000
-                if [[ $devname == 'SMBus iSMT adapter at '* ]]; then
-                        found=1
-                break
-                fi
-        done
+        # Detech 
+        ismt_bus=$(i2cdetect -l|grep 'SMBus iSMT'| cut -c5)
```

Comment:
Inconsistent spacing around pipes. Please use consistent spacing for readability.

Reply:
updated https://github.com/Azure/sonic-buildimage/pull/6889/commits/fa917e6e0605c7427d95ecdcedbb8335f05cae84